satpy/tests/reader_tests/test_aws1_mwr_l1b.py

```diff
@@ -0,0 +1,120 @@
+"""Tests for aws l1b filehandlers."""
+
+from datetime import datetime
```

Comment:
Please use `import datetime as dt` so that further imports are possible.

Reply:
Ah, yesh, food. Fixed.